### DISCUSSION TITLE
How programatically control flow?

### DISCUSSION THREAD

**Discussion by tpoxa** [bcakmakoglu/vue-flow#185] [Q&A] [answered]
Hey. Sorry, can not find a way how I can make the entire flow locked (non-interactive) programmatically.  Thanks!

**Comment by bcakmakoglu** [comment]
You can disable default change handlers by setting `applyDefault` to false, either with `useVueFlow({ applyDefault: false })` or as a prop `<VueFlow :apply-default="false" />`

It will disable handling of drag, select, removal etc., though the events are still emitted as `onNodesChange` or `onEdgesChange` and you can apply the changes manually with `applyNodeChanges` or `applyEdgeChanges` provided by `useVueFlow`

**Comment by bcakmakoglu** [comment, marked as answer]
Sorry after re-reading your question I believe what you're looking for is probably more simple than `applyDefault` 😅

You probably just want to lock interaction, so you can just use the [`setInteractive`](https://types.vueflow.dev/interfaces/Actions.html#setInteractive) action of `useVueFlow`.

This will disable/enable dragging nodes, connecting nodes and selecting elements.

**Reply by tpoxa** [reply]
Thank you
